cmake/FindZstd.cmake

```diff
@@ -2,19 +2,17 @@ find_package (PkgConfig REQUIRED)
 
 pkg_search_module (Zstd_PC
   QUIET
-  zstd)
+  libzstd)
```

Comment:
is it `liblibzstd` vs `libzstd` or is this a typo

Reply:
it's not a typo, though `pkg-config` on the cli seems to be a bit more forgiving.